python/lsst/daf/butler/registries/sql.py

```diff
@@ -415,9 +415,21 @@ def registerDatasetType(self, datasetType: DatasetType) -> bool:
         _, inserted = self._managers.datasets.register(datasetType)
         return inserted
 
-    def removeDatasetType(self, name: str) -> None:
+    def removeDatasetType(self, name: str | tuple([str, ...])) -> None:
```

Comment:
You will have to also update the base class (python/lsst/daf/butler/registry/_registry.py) to have the same call signature and to document the `name` parameter. You will likely also need to update `remote.py` in this directory.